site/examples/examplesPageStyle.less

```diff
@@ -6,6 +6,7 @@
 
```

Comment:
needed for the change to react-dimensions for responsive styling of the examples page